schemas/v1/schema.json

```diff
@@ -0,0 +1,116 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://github.com/dotnet/docfx/schemas/v1/schema.json#",
```

Comment:
>https://github.com/dotnet/docfx/schemas/v1/schema.json# [](start = 12, length = 55)

seems it's better not end with #? [https://spacetelescope.github.io/understanding-json-schema/structuring.html#the-id-property](https://spacetelescope.github.io/understanding-json-schema/structuring.html#the-id-property)